python/tvm/relay/frontend/tflite.py

```diff
@@ -325,8 +349,14 @@ def convert_softmax(self, op):
         input_tensor_idx = input_tensor.tensor_idx
         params = {'axis': 1}  # 1 is channel
         in_expr = self.get_expr(input_tensor_idx)
-        out = _op.nn.softmax(in_expr, **params)
 
+        # Softmax does not go well with Int8. So Dequantize back to FP32.
```

Comment:
Better leave one TODO. Softmax's int8 computation is complex but we could leave it to implement when we have spare time. The same situation suits for like `sigmoid` / `mean` operators.